index.js

```diff
@@ -36,6 +36,30 @@ function render(resume) {
     resume.basics.top_five_profiles = resume.basics.profiles.slice(0, 5);
     resume.basics.remaining_profiles = resume.basics.profiles.slice(5);
 
+    _(resume.projects).forEach(project_info => {
+        const start_date = moment(project_info.startDate, 'YYYY-MM-DD');
+        const end_date = moment(project_info.endDate, 'YYYY-MM-DD');
+        const can_calculate_period = start_date.isValid() && end_date.isValid();
+
+        if (can_calculate_period) {
+            project_info.duration = moment.preciseDiff(start_date, end_date);
+        }
+
+        if (start_date.isValid()) {
+          project_info.startDate = utils.getFormattedDate(start_date);
+        }
+
+        if (end_date.isValid()) {
+          project_info.endDate = utils.getFormattedDate(end_date);
+        }
+
+        project_info.name = convertMarkdown(project_info.name);
```

Comment:
IMO we shouldn't allow markdown in the project's name. Visually, it should follow the styles specified by the theme

Reply:
Sure, Makes sense